blog/2025-03-31-envio-dev-update-march-2025.md

```diff
@@ -8,15 +8,46 @@ slug: /envio-developer-update-march-2025
 
 <!--truncate-->
 
-March was madness! We’re excited to roll out HyperIndex v2.14.0 with RPC failover support, LogTui for fast chain scans, and integrations with V12, XDC Network, Monad, and Chiliz. With new tools, powerful partnerships, tutorials, and more, we’re empowering developers to build smarter and faster than ever. Let’s dive into the latest updates!
+March was madness! We’re excited to roll out HyperIndex v2.15.0 with RPC failover support, LogTui for fast chain scans, and integrations with V12, XDC Network, Monad, and Chiliz. With new tools, powerful partnerships, tutorials, and more, we’re empowering developers to build smarter and faster than ever. Let’s dive into the latest updates!
 
 
-## 🚀 HyperIndex Version 2.14.0 is now available 🚀
+## 🚀 HyperIndex Version 2.15.0 is now available 🚀
 
-<img src="/blog-assets/envio-dev-update-march-1-2025.png" alt="v2.14.0" width="100%"/>
+Topic Filtering goes Multichain ⛓️
+Now the `eventFilters` option can also accept a callback, allowing for building different filters depending on `chainId`:
```

Comment:
This changes belong to `2.14.0` which was previously correct. What I meant that there are two releases `2.14.0` with RPC fallback and `2.15.0` with multichain topic filtering.